src/deepLink/deepLink.ts

```diff
@@ -0,0 +1,101 @@
+import type { Uri } from 'vscode';
+import type { GitRemote } from '../git/models/remote';
+import type { Repository } from '../git/models/repository';
+
+export enum DeepLinkType {
+	Branch = 'b',
+	Commit = 'c',
+	Remote = 'r',
```

Comment:
Can we call this `Repository` -- I keep thinking that we are trying to target an actual remote

Reply:
Done in commit: [Update enums](https://github.com/gitkraken/vscode-gitlens/pull/2471/commits/45a4b650d2d2b719c7ee4e4e5af9c08728b47d89)